conans/client/rest/rest_client_v2.py

```diff
@@ -293,6 +294,9 @@ def remove_conanfile(self, ref):
     def get_recipe_revisions(self, ref):
         url = self.router.recipe_revisions(ref)
         tmp = self.get_json(url)["revisions"]
+        # FIXME: the server API is returning an iso date, we have to convert to timestamp
```

Comment:
Please remember to annotate this in the doc for server potential changes.

Reply:
It's already there 👍